x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_info_popover.tsx

```diff
@@ -0,0 +1,123 @@
+    infoTitle = i18n.translate('xpack.aiops.analysis.analysisTypeSpikeInfoTitle', {
+      defaultMessage: 'Log rate spike',
+    });
+    infoContent = i18n.translate('xpack.aiops.analysis.analysisTypeSpikeInfoContent', {
+      defaultMessage:
+        'The median log rate in the selected deviation time range is higher than the baseline. Therefore, the analysis results table shows statistically significant items within the deviation time range that are contributors to the spike. The "doc count" column refers to the amount of documents in the deviation time range.',
+    });
+  } else if (!zeroDocsFallback && analysisType === LOG_RATE_ANALYSIS_TYPE.DIP) {
+    infoTitle = i18n.translate('xpack.aiops.analysis.analysisTypeDipInfoTitle', {
+      defaultMessage: 'Log rate dip',
+    });
+    infoContent = i18n.translate('xpack.aiops.analysis.analysisTypeDipInfoContent', {
+      defaultMessage:
+        'The median log rate in the selected deviation time range is lower than the baseline. Therefore, the analysis results table shows statistically significant items within the baseline time range that are less in number or missing within the deviation time range. The "doc count" column refers to the amount of documents in the baseline time range.',
+    });
+  } else if (zeroDocsFallback && analysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE) {
+    infoTitle = i18n.translate('xpack.aiops.analysis.analysisTypeSpikeFallbackInfoTitle', {
+      defaultMessage: 'Top items for deviation time range',
+    });
+    infoContent = i18n.translate('xpack.aiops.analysis.analysisTypeSpikeInfoContentFallback', {
+      defaultMessage:
+        'The baseline time range does not contain any documents. Therefore the results show top log message categories and field values for the deviation time range.',
+    });
+  } else if (zeroDocsFallback && analysisType === LOG_RATE_ANALYSIS_TYPE.DIP) {
+    infoTitle = i18n.translate('xpack.aiops.analysis.analysisTypeDipFallbackInfoTitle', {
+      defaultMessage: 'Top items for baseline time range',
+    });
+    infoContent = i18n.translate('xpack.aiops.analysis.analysisTypeDipInfoContentFallback', {
+      defaultMessage:
+        'The deviation time range does not contain any documents. Therefore the results show top log message categories and field values for the baseline time range.',
+    });
+  } else {
+    return null;
+  }
+
+  return (
+    <EuiPopover
+      anchorPosition="upCenter"
+      button={
+        <LogRateAnalysisInfoPopoverButton
+          onClick={setIsPopoverOpen.bind(null, !isPopoverOpen)}
+          label={infoTitle}
+        />
+      }
+      closePopover={setIsPopoverOpen.bind(null, false)}
+      isOpen={isPopoverOpen}
+      ownFocus
+      panelPaddingSize="m"
+    >
+      {infoTitle && (
+        <EuiPopoverTitle>
+          {infoTitlePrefix}
+          {infoTitle}
+        </EuiPopoverTitle>
+      )}
+
+      <EuiText size="s" css={{ maxWidth: `calc(${euiTheme.euiSizeXL} * 15);` }}>
```

Comment:
This is a bit weird but it's a mix of what the EUI docs say to do with their sizes and what I copied over from the [ml help popover](https://github.com/elastic/kibana/blob/main/x-pack/plugins/lens/public/datasources/form_based/help_popover.tsx) while getting rid of the SCSS there.